apps/codestyle/codestyle.py

```diff
@@ -223,6 +224,10 @@ def misc_codestyle_check(file: io, file_path: str) -> None:
             print(
                 f"Please use the syntax 'Class/ObjectType const*' instead of 'const Class/ObjectType*': {file_path} at line {line_number}")
             check_failed = True
+        if ' if(' in line or ' if ( ' in line:
```

Comment:
I think it would be cleaner to make a constant list
```python
IF_SYNTAX_BREAKS = [" if(", " if ( "]
```

and then
```suggestion
        if [match for match in IF_SYNTAX_BREAKS if match in line]:
```

this way we can easily expand

Reply:
Or alternatively we switch to regex checks and make a list out of that. Then we can add

```python
IF_SYNTAX_BREAKS_REGEX = [r' if \(?.* \)', r' if\(', r' if \( ']
if [re.search(regex_check, line) for regex_check in IF_SYNTAX_BREAKS_REGEX]:
    # do stuff
```